stories/core/Footer.stories.tsx

```diff
@@ -30,21 +30,91 @@ Basic.args = {
   },
 } as FooterProps;
 
-export const Custom: Story<FooterProps> = ({
-  customElements,
+export const AppendedCustomElements: Story<FooterProps> = ({
   ...rest
 }: FooterProps) => {
-  return <Footer customElements={customElements} {...rest} />;
-};
-
-Custom.args = {
-  customElements: [
+  const customElements = [
     {
       title: 'Custom',
       url: 'https://www.bentley.com/',
     },
-  ],
-} as FooterProps;
+  ];
+  return <Footer customElements={customElements} {...rest} />;
+};
+
+export const OnlyCustomElements: Story<FooterProps> = ({
+  ...rest
+}: FooterProps) => {
+  const customElements = (defaultElements: FooterElement[]) => {
+    return [
+      {
+        title: 'Custom Element 1',
+        url: 'https://www.bentley.com/',
+      },
+      {
+        title: 'Custom Element 2',
+      },
+      {
+        title: 'Custom Element 3',
+      },
+      {
+        title: 'Custom Element 4',
+      },
+    ];
+  };
+
+  return <Footer customElements={customElements} {...rest} />;
+};
+
+export const CustomDefaultElements: Story<FooterProps> = ({
```

Comment:
Doest work for me
![image](https://user-images.githubusercontent.com/81580355/154457322-7106ab00-37b8-4274-90e7-abaca0ca0a95.png)

Maybe rename also to something like `DefaultAndCustomElements`, so its clear you are merging those

Reply:
I changed it to Customized Default And Custom Elements to differentiate it from Appended Custom Elements.